AdvLoggerPkg/Docs/ReadMe.md

```diff
@@ -267,5 +268,5 @@ result in hardware printing not functional.
 
 ## Copyright
 
-Copyright (C) Microsoft Corporation. All rights reserved.  

```

Comment:
Trailing double space in md is a new line. Do you have other alternative here? Otherwise the preview will show both lines concatenated.

Reply:
This is going to happen often because most people have editors configured to remove trailing whitespace.

I'm going to propose adding a back slash.

Source change:

```md
Copyright (C) Microsoft Corporation. All rights reserved. \
SPDX-License-Identifier: BSD-2-Clause-Patent
```

Markdown rendered change:

Copyright (C) Microsoft Corporation. All rights reserved. \
SPDX-License-Identifier: BSD-2-Clause-Patent

Reply:
Sure, let's follow that back slash pattern then.

Reply:
Added